certified-connectors/Kapish Power Connect/apiDefinition.swagger.json

```diff
@@ -0,0 +1 @@
nAddressCountry":{"type":"string","x-ms-summary":"Country"}}},"CMUpdatableLocationAddress":{"type":"object","properties":{"Uri":{"type":"integer","x-ms-summary":"Unique ID","description":"The uri of the existing street or mailing address to be updated"},"LocationAddressAddressType":{"enum":["Street","Postal"],"x-ms-enum-values":[{"value":"Street","displayName":"Street"},{"value":"Postal","displayName":"Mailing"}],"type":"string","description":"Select address type. Limit of one per type.","x-ms-summary":"Type"},"LocationAddressAddressLines":{"type":"string","x-ms-summary":"Street"},"LocationAddressCity":{"type":"string","x-ms-summary":"Suburb / City"},"LocationAddressState":{"type":"string","x-ms-summary":"State"},"LocationAddressPostcode":{"type":"string","x-ms-summary":"Postcode"},"LocationAddressCountry":{"type":"string","x-ms-summary":"Country"}}},"CMLocationAddressUri":{"title":"Child Address","type":"object","properties":{"Uri":{"type":"integer","x-ms-summary":"Unique ID"}}},"CMSetActiveDateRange":{"title":"Active","type":"object","x-ms-visibility":"advanced","properties":{"SetActiveDateRangeValidFromDate":{"type":"string","x-ms-summary":"From","description":"dd/mm/yyyy"},"SetActiveDateRangeValidToDate":{"type":"string","x-ms-summary":"To","description":"dd/mm/yyyy"}}},"CMLocationRelationship":{"x-ms-visibility":"advanced","type":"object","title":"Association","x-ms-summary":"Add Relationship","properties":{"AddRelationshipMakeThisTheDefaultRelationship":{"type":"boolean","x-ms-summary":"Make Default"},"AddRelationshipRelatedLocation":{"type":"integer","x-ms-summary":"Related Unique ID"},"AddRelationshipRelationshipType":{"type":"string","x-ms-summary":"Type","enum":["MemberOf","BossedBy","OtherOf","HasMembers","BossOf","HasOthers","DelegateOf","HasDelegates","AdministratorOf","AdministeredBy"],"x-ms-enum-values":[{"value":"MemberOf","displayName":"Member Of"},{"value":"BossedBy","displayName":"Bossed By"},{"value":"OtherOf","displayName":"Other Of"},{"value":"HasMembers","displayName":"Has Members"},{"value":"BossOf","displayName":"Boss Of"},{"value":"HasOthers","displayName":"Has Others"},{"value":"DelegateOf","displayName":"Delegate Of"},{"value":"HasDelegates","displayName":"Has Delegates"},{"value":"AdministratorOf","displayName":"Administrator Of"},{"value":"AdministeredBy","displayName":"Administered By"}]}}},"CMUploadElectronicFile":{"required":["Uri","RecordFilePath"],"type":"object","properties":{"RecordFilePath":{"type":"string","x-ms-summary":"File Name"},"Uri":{"type":"string","x-ms-summary":"Unique Identifier"}}}},"tags":[],"x-ms-connector-metadata":[{"propertyName":"Website","propertyValue":"https://kapish.com.au/products/power_connect/"},{"propertyName":"Privacy policy","propertyValue":"https://kapish.com.au/products/privacy-policy/"},{"propertyName":"Categories","propertyValue":"Data;Content and Files"}],"security":[{"API Key":[]}],"securityDefinitions":{"API Key":{"type":"apiKey","in":"header","name":"KPCSubscriptionKey"}}}
```

Comment:
@BrettHankinson please have a valid swagger.json file. it should be in json format and please fix the indentation. all files should be properly formatted. please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).

Reply:
Thanks @Amjed-Ayoub I have formatted our swagger file.